components/slideshow.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import { useState } from 'react';
+export default function Slideshow (){
```

Comment:
Are you running `npm run format`? Wonder how are changes being committed without formatting